src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -962,8 +1065,23 @@ private string InvokeReadLine(CancellationToken cancellationToken)
         private void InvokeInput(string input, CancellationToken cancellationToken)
         {
             SetBusy(true);
+
             try
             {
+                if (_shellIntegrationEnabled)
+                {
+                    InvokePSCommand(
+                        new PSCommand().AddScript("Write-Host -NoNewLine \"`e]633;C`a\""),
+                        new PowerShellExecutionOptions
+                        {
+                            AddToHistory = false,
+                            ThrowOnError = false,
+                            WriteOutputToHost = true,
+                            FromRepl = true,
+                        },
+                        cancellationToken);
```

Comment:
```suggestion
                    Console.Write("\x1b]633;C\a");
```

Optional nit, we can just directly write instead of involving the host or PowerShell at all

Reply:
Brilliant!